.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch Package",
```

Comment:
?

```suggestion
      "name": "Debug translate service",
```

Reply:
Yeah, will, this was the default for generating launch.json